_data/announcements.yml

```diff
@@ -1,5 +1,4 @@
 # Announcements
-- announcement: "A pre-release of the Zowe SMP/E install is now available. This is an alpha release based on Zowe 1.8.0, and should not be used in production."
-  link: "https://docs.zowe.org/stable/getting-started/summaryofchanges.html#zowe-smp-e-alpha-august-2019"
+- announcement: "The first release of the Zowe SMP/E FMID is now available. This is a release based on Zowe 1.9.0."
```

Comment:
"The first release of the Zowe SMP/E build is now available, with FMID of AZWE001.  This is the same content as the Zowe 1.9.0 convenience build!.

Reply:
I updated the announcement with your suggestion, thanks!